components/api/src/main/java/com/hotels/styx/api/HttpHeader.java

```diff
@@ -44,7 +43,7 @@ public static HttpHeader header(String name, String... values) {
             throw new IllegalArgumentException("must give at least one value");
         }
 
-        return new HttpHeader(requireNonNull(name), ImmutableList.copyOf(values));
+        return new HttpHeader(requireNonNull(name), listOf(values.clone()));
```

Comment:
Why is it necessary to clone the values?

Reply:
I think that I originally had an implementation of `listOf` that was based on `Arrays.asList()` (or perhaps I originally just used that construction here directly) - and that creates an `ArrayList` that is backed by the original array. Which is different from `ImmutableList.copyOf()`. So the `clone` is needed for that. But, with the current implementation, that's not necessary - I'll remove it.